### ISSUE TITLE
Bug: Windows text selection shortcut does not work in search input

### ISSUE THREAD

**Issue by brianpeiris** [mdn/yari#5482]
# Summary

The suggested result functionality in the search input seems to interfere with Windows text selection keyboard shortcuts.

## Where did you spot the bug?

This happens with the search input on the MDN home page, and the search input on reference pages.

## What is the problem?

When you type some text into the search input, I would expect to be able to use typical Window text selection shortcuts. In particular, I have a habit of using `Shift + Home` to select existing text in a text input in order to edit or delete it. This works everywhere in Windows, but it looks like MDN's search input uses `Home` to move to the top of the suggested results list, which overrides the default behavior of `Shift + Home`

Arguably, it would also be nice not to interfere with other typical text operation shortcuts (`Home`, `End`, `Shift + Home`, `Shift + End`, etc) and use extended modifiers (`Alt + Home` or `Shift + Alt + Home`) for the suggested results list instead.

## Steps to reproduce the issue

1. On Windows, visit a reference page like https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog
2. Focus the search input on the page
3. Type some text in the search input
4. Type `Shift + Home`
5. See that the text does not get selected

### Device

- [x] Desktop

### Browser

- [x] Chrome: (98)
- [x] Firefox: (97, 99)

### Operating system

- [x] Windows: (Windows 10)

**Comment by caugner** [comment]
Hi @brianpeiris, thank you for taking the time to open this issue. 👋 

Please note that we're using [Downshift.js](https://github.com/downshift-js/downshift) and their [Combobox](https://www.downshift-js.com/use-combobox/) implementation for the search suggestions, and we haven't customized their [default event handlers](https://github.com/downshift-js/downshift/tree/1c7698080805e05f5180dd223cbc17932a85fd42/src/hooks/useCombobox#event-handlers), nor do we plan to do so.

However, you should be able to use `Ctrl+LeftArrow` and `Ctrl+RightArrow` to go to the start / end of the input value. 😉 

PS: This has meanwhile been reported again in https://github.com/mdn/yari/issues/6466.
